cmd/minishift/cmd/delete.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/docker/machine/libmachine"
 	"github.com/minishift/minishift/pkg/minikube/cluster"
 	"github.com/minishift/minishift/pkg/minikube/constants"
+	instanceConfig "github.com/minishift/minishift/pkg/minishift/config"
```

Comment:
Using `instanceConfig` every where to refer `pkg/minishift/config` since we have other config dirs as well.